src/index.js

```diff
@@ -313,16 +317,32 @@ async function main(req, context) {
     AWS_REGION: region, AWS_ACCOUNT_ID: accountId, ...env, owner, repo, ref,
   }, log);
 
+  const method = req.method.toUpperCase();
   let responses;
-  if (change.deleted) {
+
+  if (method === 'GET') {
     responses = await Promise.all(indices.map(
-      async (index) => ({ name: index.config.name, ...await handleDelete(index, change, log) }),
+      async (index) => (
+        { name: index.config.name, ...await handleStatus(index, change, log) }
+      ),
     ));
```

Comment:
This new behaviour on `GET` is backwards compatible: the invocation triggered by SQS is converted to a `POST` in `./records-wrapper.js`